dataprofiler/data_readers/filepath_or_buffer.py

```diff
@@ -0,0 +1,45 @@
+from io import open, StringIO, BytesIO
```

Comment:
for now, i suggest moving it to utils or maybe we should rename as I'm not sure this should be its own file. I'll wait for consensus from others...
@granteden @ChrisWallace2020 @AnhTruong ?

Reply:
I'm inclined to merge this and fix in following PR if needed.